### ISSUE TITLE
Invisible buttons on the grid/list popup menu

### ISSUE THREAD

**Issue by Lllum** [fast4x/RiMusic#2101]
### Steps to reproduce the bug

Tap and hold a song that isn't in your Playlist or tap the three dots in the top right corner to view the grid/list layout with the add to Playlist/add to queue/play next/etc... (only happens with songs that aren't added to a Playlist) 

### Expected behavior

The full layout should be visible, the following imagine is with a song that's already been added to a Playlist before. 

![Screenshot_20240531-021908](https://github.com/fast4x/RiMusic/assets/156969262/2006eb02-f4cd-4683-9e61-5716c4fd88c2)

### Actual behavior

Almost half of the popup for grid/list is is invisible, the following screenshot is the same screen but with a song I don't have in my Playlist. 

![Screenshot_20240531-021901](https://github.com/fast4x/RiMusic/assets/156969262/f0915d2f-ff11-4bb2-a3c1-0dc7b8d727e3)

And here, it does the same on the player screen as well. 

![Screenshot_20240531-021919](https://github.com/fast4x/RiMusic/assets/156969262/94e91075-65db-4c41-af7b-fe065320cf7d)

### RiMusic version

0.6.36.1

### Android version

8

### Additional information

Tapping on the empty space, the buttons work, I just cant see them

**Comment by fast4x** [comment]
check new version if fixed

**Comment by Lllum** [comment]
> check new version if fixed

Nope, it's still like that, and when I click add to Playlist it doesn't show me the list of Playlists, just dims the screen a little

**Comment by fast4x** [comment]
ok I'll check it better

**Comment by Lllum** [comment]
> ok I'll check it better 

Hi, I found a new area where this bug is present. When I search up a song I can only see the cover, like button, and download button.

 ![Screenshot_20240602-175309.jpg](https://github.com/fast4x/RiMusic/assets/156969262/c8855146-23d5-4e57-a649-05295119f25e)
